packages/devtools_app/test/typed_data_variable_test.dart

```diff
@@ -69,7 +70,8 @@ void main() {
       ),
       isolateRef,
     );
-    when(manager.service.getObject(any, any, offset: 0, count: 4))
+
+    when(manager.service!.getObject('1', '123', offset: 0, count: 4))
```

Comment:
can you comment on whether '1' and '123' match some test data somewhere or if they are just arbitrary?

Reply:
I replaced them with constants to make them more trackable. 
I am not sure it makes sense to add comments, as there are many occurrences. Mockito documentation makes it clear that values should match the data. If it is not clear enough, it would be better to pull request the documentation, rather then comment it in code:

https://github.com/dart-lang/mockito/blob/master/NULL_SAFETY_README.md#mocking-members-with-non-nullable-type-variable-return-types

Reply:
replacing with consts is good. Sometimes it is just helpful to have a simple comment "These values are arbitrary" where the consts are defined to help other people reading the code to know that they are not "special" values. In other words, that these values can be changed without breaking things.